core/storage/exploration/gae_models.py

```diff
@@ -497,105 +497,3 @@ def get_recently_published(cls, limit):
         ).order(
             -ExpSummaryModel.first_published_msec
         ).fetch(limit)
-
-
-class StateIdMappingModel(base_models.BaseModel):
```

Comment:
Should we deprecate this class for now instead of removing it altogether? We have some entities in the datastore, it may be better not to remove their definition until they're actually cleared out.

Reply:
That's a good point. I'll do that.